test/58-assert-10-assert_equal_files.bats

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bats
+
+load 'test_helper'
+fixtures 'exist'
+
+# Correctness
+@test 'assert_equal_files() <file>: returns 0 if <file1> and <file2> are the same' {
+  local -r file1="${TEST_FIXTURE_ROOT}/dir/file_with_text"
+  local -r file2="${TEST_FIXTURE_ROOT}/dir/same_file_with_text"
+  run assert_equal_files "$file1" "$file2"
+  [ "$status" -eq 0 ]
+  [ "${#lines[@]}" -eq 0 ]
+}
+
+
+@test 'assert_equal_files() <file>: returns 1 if <file1> and <file2> are not the same' {
+  local -r file1="${TEST_FIXTURE_ROOT}/dir/same_file_with_text"
+  local -r file2="${TEST_FIXTURE_ROOT}/dir/file"
+  run assert_equal_files "$file1" "$file2"
+  [ "$status" -eq 1 ]
+  [ "${#lines[@]}" -eq 3 ]
+  [ "${lines[0]}" == '-- files are not the same --' ]
+  [ "${lines[1]}" == "path : $file" ]
```

Comment:
In the README you describe that the output shows `path1` and `path2` but here it's only `path`
IMO the output should show 
```
file1 : $file1
file2 : $file2
```